composer.json

```diff
@@ -24,7 +24,7 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
+        "php": "^7.2.0",
```

Comment:
Would it make sense to use 7.2.9 like Symfony 5.0?

Reply:
sure, updated.